src/test/java/hudson/plugins/buildblocker/PipelineJobListenerTest.java

```diff
@@ -0,0 +1,28 @@
+package hudson.plugins.buildblocker;
+
+import hudson.model.FreeStyleProject;
+import org.junit.Rule;
+import org.junit.Test;
+
+import java.io.IOException;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class PipelineJobListenerTest {
+
+    @Rule
+    public JenkinsRule jenkins = new JenkinsRule();
+
+    @Test
+    public void should_add_blocking_properties_from_config_file() throws IOException {
+        FreeStyleProject job1 = jenkins.createFreeStyleProject("testJob");
```

Comment:
maybe I will call `testJob` instead of `job1`

Reply:
better, fixed it.